apps/base-docs/src/theme/NavbarItem/ComponentTypes.js

```diff
@@ -32,6 +32,7 @@ export const CustomConnectButton = ({ className }) => {
                 action: ActionType.change,
                 context: 'navbar',
                 address,
+                walletType: connector?.name.toLowerCase().replace(/[ \-\/\.]/g, '_'),
```

Comment:
minor: this could be a function so that we don't have the same logic (`.toLowerCase().replace(/[ \-\/\.]/g, '_')` twice in two files

Reply:
good callout. added a helper function in libs for this.